.rubocop_todo.yml

```diff
@@ -14,7 +14,12 @@ Metrics/AbcSize:
 # Configuration parameters: CountComments, ExcludedMethods.
 # ExcludedMethods: refine
 Metrics/BlockLength:
-  Max: 89
+  Max: 50
```

Comment:
Should we not move this into `.rubocop.yml`? The idea is to eventually get rid of `.rubocop_todo.yml` , either by refactoring code to not require any non-default configuration or by setting some reasonable project specific configuration in  `.rubocop.yml`.

Reply:
Moved.